### PR TITLE
Updated Documentation

### PR DESCRIPTION
**Pull request by orifox**
Replace MD with RST since I wanted to add a table.  Also, slight changed to documentation.

### CODE REVIEW

**Review by eteq — CHANGES_REQUESTED** [review]
Hmm, so am I understanding right that you switched to RST just for a table?  markdown *does* support tables sort of like RST - for example, 

```
| Syntax      | Description |
| ----------- | ----------- |
| Header      | Title       |
| Paragraph   | Text        |
```
renders as:
| Syntax      | Description |
| ----------- | ----------- |
| Header      | Title       |
| Paragraph   | Text        |

So if it is just for that, then maybe it's better to just use the md-style table given that most of the other material in this repo is markdown (i.e., the notebooks themselves). But not a big deal.

But even if you want to keep the rst version, you'll need to delete the `README.md` !  This PR seems to just add the rst version without deleting the old one.

File: README.md [hunk 1]
```diff
@@ -1,27 +1,42 @@
-# James Webb Space Telescope Data Analysis Tool Notebooks
+James Webb Space Telescope Data Analysis Tool Notebooks
+--------------
```
Comment (eteq) [inline README.md:2]:
```suggestion
# James Webb Space Telescope Data Analysis Tool Notebooks
```
Why this change? the `# XXX` is markdown style for headers, and since this file's name is `.md`, it should follow markdown format.

Or did you mean to produce a horizontal line?  If so, you probably want this:
```suggestion
# James Webb Space Telescope Data Analysis Tool Notebooks
--------------
```
Since you still want it to be in header style

File: README.md [hunk 2]
```diff
@@ -1,27 +1,42 @@
...
+The following [table](https://spacetelescope.github.io/jdat_notebooks/) summarizes the notebooks currently available.
 
- The notebooks attempt to utilize a number of software packages supported by STScI, including [Astropy](https://www.astropy.org), [glue](http://docs.glueviz.org/en/stable/index.html), [ginga](https://ginga.readthedocs.io/en/latest/), [photutils](https://photutils.readthedocs.io), [specutils](https://specutils.readthedocs.io/en/stable/), [astroimtools](http://astroimtools.readthedocs.io), [imexam](http://imexam.readthedocs.io), [jdaviz](https://jdaviz.readthedocs.io/en/latest/), [asdf](http://asdf.readthedocs.io/en/latest/), [gwcs](https://gwcs.readthedocs.io/en/latest/), and [synphot](http://synphot.readthedocs.io/en/latest/index.html).  Note jdaviz is STScI's JWST Data Analysis Visualization Tool, designed to be used with spectra, IFU cubes, and multi-object spectroscopy (MOS).
+Installation
+--------------
```
Comment (eteq) [inline README.md:9]: As above, should be
```suggestion
# Installation
```
or
```suggestion
# Installation
--------------
```

File: README.md [hunk 3]
```diff
@@ -1,27 +1,42 @@
...
+<pre><code>   git branch -m master main
+   git fetch origin
+   git branch -u origin/main main
+</code></pre>
+
 If you are using a GUI to manage your repos you'll have to find the equivalent commands as it's different for different programs. Alternatively, you can just delete your local clone and re-clone!
+
+
+Help
+----------
```
Comment (eteq) [inline README.md:32]:
```suggestion
# Help
----------
```
or
```suggestion
# Help
```

File: README.md [hunk 4]
```diff
@@ -1,27 +1,42 @@
...
+
+If you uncover any issues or bugs, you can open a GitHub ticket.  For faster responses, however, we encourage you to submit a JWST Help Desk Ticket: jwsthelp.stsci.edu
+
+Contributing
+----------
```
Comment (eteq) [inline README.md:37]:
```suggestion
# Contributing
----------
```
or
```suggestion
# Contributing
```

**Review by eteq — CHANGES_REQUESTED** [review]
Some fixes to match .md .  *however* I'm realizing I'm not sure what your intent was here - are you trying to switch to rst format?

**Review by eteq — APPROVED** [review]

**Comment by eteq** [comment]
There was an unintended commit mixed in here but I fixed it with a rebase.  I think this is now ready to go once the CI is done!

**Comment by orifox** [comment]
Two CI's fail, but merging anyway. The "PR" one is a timeout that I think showed up in a notebook last week.  The publish one is about some security keys that may need fixing but is not affected by this PR since it doesn't touch the relevant files at all.
